ONSdigital/ssdc-rm-notify-service
# Order the OpenAPI docs

# Motivation and Context
The OpenAPI docs were juggling the order of things around every time the machine generation process was happening, resulting in 'changes' which weren't really changes at all.

# What has changed
Used a setting to control the order, so it's consistent.

# How to test?
Build the project a load of times and check that api.json and api.md don't change at all.

# Links
Trello: https://trello.com/c/lx1m9L3M
